### DISCUSSION TITLE
Error Log 'files archive' and 'passwords'

### DISCUSSION THREAD

**Discussion by GUNT0815** [nextcloud/all-in-one#4456] [Questions] [answered]
How do I fix the following errors that occure multiple times and ongoing in my error log of my nextcloud-aio installation?

1) [files_archive] Error: /var/www/html/custom_apps/files_archive/lib/Listener/FilesActionListener.php:179: OCA\FilesArchive\Listener\FilesActionListener::handle: Unable to add style asset files-sidebar-hooks
	GET /apps/files/

2) [passwords] Fatal: Exception::__construct(): Argument #2 ($code) must be of type int, GuzzleHttp\Exception\ClientException given
	GET /index.php/apps/passwords/api/1.0/service/favicon/eisencloud.de/32

**Comment by szaimen** [comment, marked as answer]
Hi, this needs to be fixed in the corresponding apps

**Comment by BraidenPsiuk** [comment]
I also noticed tons of logs from the `Passwords` app in my AIO instance over the past week or two. They all seemed to stem from the favicon service I was using. In my case simply flushing the caches seemed to help reduce most of the logs, IIRC it at least got rid of the "GuzzleHttp" errors.
```
https://your-nextcloud.example.com/settings/admin/passwords --> "Caches"
```
But to be safe, I also did an export of my passwords, reset the app, re-imported my passwords, and set "Favicon Service" to "none" for the time being. Haven't noticed any errors after an hour or so since I did that.

**Reply by BraidenPsiuk** [reply]
Never mind- doesn't matter, the GuzzleHttp errors are back... Just took around 2 or 3 hours apparently :) Maybe changing "Password Security Checks" to one of the local databases will prevent it if it's due to periodically checking HIBP. But personally I'm just leaving my settings untouched and ignoring that error until it's properly fixed.
